### ISSUE TITLE
mediasoup-worker crashed 

### ISSUE THREAD

**Issue by magiczhangx** [versatica/mediasoup#1056]
## Bug Report

Mediasoup-worker crashed.  Attached core file.
### Core file:
[core.zip](https://github.com/versatica/mediasoup/files/11176080/core.zip)

### environment

- Operating system: ubuntu:20.04 (Linux 3.10.0-693.2.2.el7.x86_64 1 SMP Tue Sep 12 22:26:13 UTC 2017 x86_64 x86_64 x86_64 GNU/Linux)
- Node version: v19.8.1
- npm version: 9.5.1
- gcc/clang version: gcc version 9.4.0 (Ubuntu 9.4.0-1ubuntu1~20.04.1)
- mediasoup version: 3.11.19
- mediasoup-client version:  mediasoup-client(v3.6.69) , libmediasoupclient(v3.3.0)

#### Scenario:

- peer1: web peer with mediasoup-client v3.6.69
- peer2: windows peer with libmediasoupclient v3.3.0
- all peers produce one video stream and one audio stream and comsume others' streams.
- after scenario run some hours , worker crashed.

#### Server:
using below apis(simplified) to start up service
```javascript
const worker = mediaSoup.createWorker({
    logLevel: "warn",
})

webrtcServer = await worker.createWebRtcServer({
    listenInfos: [
        {
            protocol: "udp",
            ip: "......my ip",
            port: 7000,
        },
    ],
});

const mediaCodecs = [
    {
        kind: "audio" as const,
        // name: "opus",
        mimeType: "audio/opus",
        clockRate: 48000,
        channels: 2,
    },
    {
        kind: "video" as const,
        // name: "VP8",
        mimeType: "video/VP8",
        clockRate: 90000,
        rtcpFeedback: [{ type: "nack" }, { type: "nack", parameter: "pli" }, { type: "ccm", parameter: "fir" }, { type: "goog-remb" }, { type: "transport-cc" }],
    },
    {
        kind: "video" as const,
        // name: "VP9",
        mimeType: "video/VP9",
        clockRate: 90000,
        rtcpFeedback: [{ type: "nack" }, { type: "nack", parameter: "pli" }, { type: "ccm", parameter: "fir" }, { type: "goog-remb" }, { type: "transport-cc" }],
    },
];

let router = await worker.createRouter({
    mediaCodecs,
});

```

**Comment by magiczhangx** [comment]
```
(gdb) bt full 
#0  __GI_raise (sig=sig@entry=6) at ../sysdeps/unix/sysv/linux/raise.c:50
        set = {__val = {0, 94865450681762, 17651987460000, 1200, 0, 18446744073709551615, 18446744073709551615, 94865450662725, 94865483443968, 3951135908071957760, 140724147486656, 94865450651504, 
            18446744073709551615, 17651987460, 44776, 1879784}}
        pid = <optimized out>
        tid = <optimized out>
        ret = <optimized out>
#1  0x00007faac9a20859 in __GI_abort () at abort.c:79
        save_stage = 1
        act = {__sigaction_handler = {sa_handler = 0x1e29cd7fe9399800, sa_sigaction = 0x1e29cd7fe9399800}, sa_mask = {__val = {2491767270827615662, 1750185709636347392, 10823973506909622734, 10712564946324113468, 
              94865447566168, 16787365640450711513, 0, 0, 0, 0, 140371503720680, 10, 94865450189114, 94865483755616, 140371505988002, 3086873673822963045}}, sa_flags = -1754098960, sa_restorer = 0x7ffce4d280b0}
        sigs = {__val = {32, 0 <repeats 15 times>}}
#2  0x00007faac9a8b26e in __libc_message (action=action@entry=do_abort, fmt=fmt@entry=0x7faac9bb5298 "%s\n") at ../sysdeps/posix/libc_fatal.c:155
        ap = {{gp_offset = 24, fp_offset = 22087, overflow_arg_area = 0x7ffce4d28180, reg_save_area = 0x7ffce4d28110}}
        fd = <optimized out>
        list = <optimized out>
        nlist = <optimized out>
        cp = <optimized out>
#3  0x00007faac9a932fc in malloc_printerr (str=str@entry=0x7faac9bb71e0 "munmap_chunk(): invalid pointer") at malloc.c:5347
No locals.
#4  0x00007faac9a9354c in munmap_chunk (p=<optimized out>) at malloc.c:2830
        pagesize = <optimized out>
        size = <optimized out>
        __PRETTY_FUNCTION__ = "munmap_chunk"
        mem = <optimized out>
        block = <optimized out>
        total_size = <optimized out>
#5  0x0000564795405799 in RTC::SeqManager<unsigned short, (unsigned char)0>::ClearDropped() ()
No symbol table info available.
#6  0x00005647954058c6 in RTC::SeqManager<unsigned short, (unsigned char)0>::Input(unsigned short, unsigned short&) ()
No symbol table info available.
#7  0x000056479540ebcf in RTC::SimulcastConsumer::SendRtpPacket(RTC::RtpPacket*, std::shared_ptr<RTC::RtpPacket>&) ()
No symbol table info available.
#8  0x00005647953e4978 in RTC::Router::OnTransportProducerRtpPacketReceived(RTC::Transport*, RTC::Producer*, RTC::RtpPacket*) ()
No symbol table info available.
#9  0x00005647953d0f4d in RTC::Producer::ReceiveRtpPacket(RTC::RtpPacket*) ()
No symbol table info available.
#10 0x000056479541ca37 in RTC::Transport::ReceiveRtpPacket(RTC::RtpPacket*) ()
No symbol table info available.
#11 0x00005647954466c4 in RTC::WebRtcTransport::OnRtpDataReceived(RTC::TransportTuple*, unsigned char const*, unsigned long) ()
No symbol table info available.
#12 0x000056479543a980 in RTC::WebRtcServer::OnUdpSocketPacketReceived(RTC::UdpSocket*, unsigned char const*, unsigned long, sockaddr const*) ()
No symbol table info available.
#13 0x00005647957477d8 in uv.udp_recvmmsg ()
No symbol table info available.
#14 0x0000564795748543 in uv.udp_io ()
No symbol table info available.
#15 0x000056479574bef5 in uv.io_poll ()
No symbol table info available.
#16 0x000056479573e52d in uv_run ()
No symbol table info available.
#17 0x000056479534778d in DepLibUV::RunLoop() ()
No symbol table info available.
#18 0x0000564795351ceb in Worker::Worker(Channel::ChannelSocket*, PayloadChannel::PayloadChannelSocket*) ()
No symbol table info available.
#19 0x0000564795346587 in mediasoup_worker_run ()
--Type <RET> for more, q to quit, c to continue without paging--
No symbol table info available.
#20 0x000056479534525f in main ()
No symbol table info available.

```

**Comment by ibc** [comment]
That crash was fixed in 3.11.20, see CHANGELOG.md.
